src/main/java/com/google/cloud/spanner/pgadapter/metadata/OptionsMetadata.java

```diff
@@ -230,6 +231,12 @@ public Builder setCredentials(Credentials credentials) {
       return this;
     }
 
+    /** Disables internal retries of aborted read/write transactions. */
+    public Builder setDisableInternalRetries() {
```

Comment:
We should remove this from this pull request. If it is something that we want for this benchmark, or in general, then it should be in a separate PR, so that this PR only contains changes related to the actual benchmark.

Reply:
Removed.